packages/web-components/src/components/va-table/va-table-inner/va-table-inner.scss

```diff
@@ -76,6 +74,16 @@ caption {
   font-size: 1.25rem;
   font-family: var(--font-serif);
   margin-bottom: 0.75rem;
+  margin-top: 0.75rem;
+  margin-left: 0.25rem;
+  margin-right: 0.25rem;
+  span {
+    display: block;
+    font-weight: normal;
```

Comment:
Hi again, @department-of-veterans-affairs/platform-design-system-designers reviewer! 👋🏼

We have an opportunity here to style caption summary however we'd like. This is my best guess. Let me know if you would prefer something else.

<img width="235" height="105" alt="Image" src="https://github.com/user-attachments/assets/10bf1954-3e59-47c8-82fb-fd80fd2eb37c" />